lib/model/model.go

```diff
@@ -1488,10 +1488,7 @@ func (m *model) ccHandleFolders(folders []protocol.Folder, deviceCfg config.Devi
 
 		// Handle indexes
 
-		if !folder.DisableTempIndexes {
-			tempIndexFolders = append(tempIndexFolders, folder.ID)
-		}
-
+		tempIndexFolders = append(tempIndexFolders, folder.ID)
```

Comment:
I'd probably still skip this for receive-encrypted folders - as we still ignore ignore progress updates on the receiving end it seems a bit pointless to track/send them.